modules/tiff/package.json

```diff
@@ -0,0 +1,49 @@
+{
+  "name": "@loaders.gl/tiff",
+  "version": "3.0.0-alpha.18",
+  "description": "Framework-independent loaders for tiff and geotiff",
+  "license": "MIT",
+  "publishConfig": {
+    "access": "public"
+  },
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/visgl/loaders.gl"
+  },
+  "keywords": [
+    "webgl",
+    "loader",
+    "3d",
+    "texture",
+    "tiff",
+    "geotiff"
+  ],
+  "types": "src/index.d.ts",
+  "main": "dist/es5/index.js",
+  "module": "dist/esm/index.js",
+  "sideEffects": false,
+  "files": [
+    "src",
+    "dist",
+    "bin",
+    "README.md"
+  ],
+  "scripts": {
+    "pre-build": "npm run build-bundle",
+    "build-bundle": "webpack --display=minimal --config ../../scripts/webpack/bundle.js"
+  },
+  "dependencies": {
+    "@loaders.gl/core": "3.0.0-alpha.18",
+    "@loaders.gl/images": "3.0.0-alpha.18",
+    "@loaders.gl/loader-utils": "3.0.0-alpha.18",
+    "@loaders.gl/worker-utils": "3.0.0-alpha.18",
+    "fast-xml-parser": "^3.16.0",
+    "geotiff": "ilan-gold/geotiff.js#ilan-gold/viv_094",
+    "math.gl": "^3.3.0",
+    "zarr": "^0.4.0"
```

Comment:
Or if we're making two new modules, `@loaders.gl/zarr` and `@loaders.gl/tiff`, then can remove the `zarr` dependency here.

Reply:
If separate modules, each should only have `zarr` and `geotiff` as dependencies respectively (`fast-xml-parser` should be removed when we delete OME-XML metadata parsing of the tiff header).

Reply:
Yep - Cleaned up the dependencies